static/app/views/performance/table.tsx

```diff
@@ -54,7 +54,7 @@ type Props = {
   organization: Organization;
   location: Location;
   setError: (msg: string | undefined) => void;
-  summaryConditions: string;
+  summaryConditions?: string;
```

Comment:
Should this remain required? It's only optional on the exported `Table` component but the inner `_Table` should require it right?